packages/kamino-sdk/src/Kamino.ts

```diff
@@ -771,6 +786,80 @@ export class Kamino {
     );
   };
 
+  getFieldsForAutodriftRebalanceMethod = async (
+    dex: Dex,
+    fieldOverrides: RebalanceFieldInfo[],
+    tokenAMint: PublicKey,
+    tokenBMint: PublicKey,
+    poolPrice?: Decimal
+  ): Promise<RebalanceFieldInfo[]> => {
+    const tokenADecimals = await getMintDecimals(this._connection, tokenAMint);
+    const tokenBDecimals = await getMintDecimals(this._connection, tokenBMint);
+    const price = poolPrice ? poolPrice : new Decimal(await this.getPriceForPair(dex, tokenAMint, tokenBMint));
+
+    // TODO: maybe we will need to get real staking price instead of pool price for this to be accurate.
```

Comment:
agree, the price received from `getPriceForPair` won't be mega accurate for what we need